metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var defaultMillisecondsDistribution = view.Distribution(0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 50000, 100000)
+
+var (
+	TaskNS, _ = tag.NewKey("namespace")
```

Comment:
Why generic name? We might consider naming it for the app instead?
```suggestion
	TaskNS, _ = tag.NewKey("visor")
```